src/typingsInstall.ts

```diff
@@ -0,0 +1,12 @@
+const spawn: any = require('cross-spawn');
+
+export default async function () {
+	return new Promise((resolve, reject) => {
+		spawn('typings', ['install'], { stdio: 'inherit' })
```

Comment:
i'll take a look then, just think it adds extra size to the command when we're suggesting that they should have it installed globally